menu/cakes/cakeA.html

```diff
@@ -0,0 +1,409 @@
+              <!-- #TODO - ENTER PRICE BELOW -->
+              <!-- <p>$ X.XX</p> -->
+            </header>
+            <h2>Description:</h2>
+            <p>Aenean ornare velit lacus, ac varius enim ullamcorper eu. Proin aliquam facilisis ante interdum congue.
+              Integer mollis, nisl amet convallis, porttitor magna ullamcorper, amet egestas mauris. Ut magna finibus
+              nisi nec lacinia. Nam maximus erat id euismod egestas. Pellentesque sapien ac quam. Lorem ipsum dolor sit
+              nullam.</p>
+            <div>
+              <h3>Available Sizes:</h3>
+              <ul>
+                <li>4-inch (8 servings)</li>
+                <li>6-inch (12 servings)</li>
+                <li>8-inch (24 servings)</li>
+                <li>10-inch (34 servings)</li>
+              </ul>
+            </div>
+            <form class="orderForm" action="https://formspree.io/f/xvoepdpy" method="POST">
+            
+              <div>
+                <h3>Select Your Options:</h3>
+                
+                <!-- Pass Cake Name to Form -->
+                <label class="hidden">
+                  <input type="text" name="cakeName" value="Cake A" readonly>
+                </label>
+
+                <label class="hidden">
+                  What would you like?
+                  <select class="cakeOrCupcakes" name="cake or cupcakes">
+                    <option value="">- Choose one -</option>
+                    <option value="cake" selected>Cake</option>
+                    <option value="cupcakes">Cupcakes</option>
+                  </select>
+                </label>
+            
+                <label class="cakeSize" style="display: none;">
+                  Size of the cake
+                  <select class="cakeSizeSelect" name="cake size">
+                    <option value="">- Choose one -</option>
+                    <option value="4in">4-inch (8 servings)</option>
+                    <option value="6in">6-inch (12 servings)</option>
+                    <option value="8in">8-inch (24 servings)</option>
+                    <option value="10in">10-inch (34 servings)</option>
+                  </select>
+                </label>
+            
+                <label class="cupcakesQuantity" style="display: none;">
+                  Quantity of the cupcakes
+                  <select class="cupcakesQuantitySelect" name="cupcakes quantity">
+                    <option value="">- Choose one -</option>
+                    <option value="6">6</option>
+                    <option value="12">12</option>
+                  </select>
+                </label>
+            
+                <label>
+                  When do you need the cake by?
+                  <input type="date" name="date" id="orderNeededDate" required>
+                </label>
+            
+                <label>
+                  Add a Custom Inscription? <i>Limit 40 chars</i> <!-- #TODO - ADD CHARACTER LIMIT -->
```

Comment:
you can remove this TODO comment since it looks like you already added the characters limit :)